documentation/asciidoc/stories/assembly_getting_started.adoc

```diff
@@ -2,7 +2,6 @@
 :context: starter_intro
 
 include::{topics}/con_sb_starter.adoc[leveloffset=+1]
-include::{topics}/proc_clone_repo.adoc[leveloffset=+1]
```

Comment:
@pruivo hey, thinking about the repo clone part for community. I put this in originally because the docs are on infinispan.org and it seemed like it would be useful to give users a way to get the code without having to search too much. see here: https://infinispan.org/infinispan-spring-boot/master/spring_boot_starter.html 

with that in mind, do you think we should remove that part or not?

Reply:
ok